src/Queue.php

```diff
@@ -155,14 +152,7 @@ public function get(array $query, $runningResetDuration, $waitDurationInMillis =
             $pollDurationInMillis = 0;
         }
 
-        //reset stuck messages
-        $this->collection->update(
-            ['running' => true, 'resetTimestamp' => ['$lte' => new \MongoDate()]],
-            ['$set' => ['running' => false]],
-            ['multiple' => true]
```

Comment:
There will be no more updating the entire collection. The running flag is removed.